.github/workflows/build.yml

```diff
@@ -149,6 +138,11 @@ jobs:
                     restore-keys: |
                         ${{ runner.os }}-node-${{ matrix.node }}-yarn-
 
+            -
+                name: Copy package.json.dist to package.json
```

Comment:
We have a solution for this - aliasing in webpack, used in other PRs. Could you adjust the PR to that thing?

Reply:
Adjusted to alias resolve in next commit.